src/main/java/uk/gov/indexer/fetchers/FetchResult.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.indexer.fetchers;
+
+import uk.gov.indexer.ctserver.SignedTreeHead;
+
+public class FetchResult {
+    private final SignedTreeHead signedTreeHead;
+    private final EntryGetter entriesFn;
```

Comment:
This now looks odd (and the associated getEntriesFn() property) given the rename to EntriesGetter.